### PR TITLE
Clarify that get_body takes into account light travel time

### PR DESCRIPTION
**Pull request by dstansby**
This adds a clarification to the docs of `get_body`.

### CODE REVIEW

File: astropy/coordinates/solar_system.py [hunk 1]
```diff
@@ -455,6 +455,8 @@ def get_body(body, time, location=None, ephemeris=None):
 
     Notes
     -----
+    The coordinate returned is the apparent position, corrected for the
+    light-travel time to the object.
```
Comment (ayshih) [inline astropy/coordinates/solar_system.py:459]:
```suggestion
    light-travel time to the location on Earth.
```

**Comment by pllim** [comment]
Needs a rebase to get rid of doc build failure.

**Comment by dstansby** [comment]
Is there any way to restart the docs build? It looks like it passed fine, but RTD had a hiccup.

**Comment by pllim** [comment]
@dstansby , unfortunately, RTD PR builder has yet to provide us with options to cancel or restart PR builds. Your best bet is a rebase (if applicable) or something that results in a new push event.

**Comment by dstansby** [comment]
🎉  it passed

**Comment by dstansby** [comment]
How about now?

**Comment by dstansby** [comment]
Sorry it took me so long to get back to this; how about now?

**Review by StuartLittlefair — APPROVED** [review]

**Comment by StuartLittlefair** [comment]
I think this looks fine now.

**Review by adrn — APPROVED** [review]
@dstansby Sorry for the long delay here, and thanks very much for the contribution! This looks good to me too.

**Pull request merged**